itests/marathon_python.feature

```diff
@@ -2,6 +2,7 @@ Feature: marathon-python can create and list marathon apps
 
   Scenario: Trivial apps can be deployed
     Given a working marathon instance
+    Then we get the marathon instance's info
```

Comment:
I think it would be more readable if this was a separate scenario (especially since this doesn't really have much to do with "Trivial apps can be deployed").